docs/topics/testing/advanced.txt

```diff
@@ -659,11 +662,15 @@ Methods
 
     Returns a ``TestSuite`` instance ready to be run.
 
-.. method:: DiscoverRunner.setup_databases(**kwargs)
+.. method:: DiscoverRunner.setup_databases(verbosity, interactive, *, **kwargs)
```

Comment:
`DiscoverRunner.setup_databases()` signature has not been changed, please revert this and a `versionadded` annotation.

Reply:
Yep, good. 👍